### PR TITLE
Added @keyframes & @supports snippet. Alphabetized

### PR DESCRIPTION
**Pull request by stowball**
@keyframes is popular and @supports is growing in usage, so snippets for them are very useful. Also alphabetized

### CODE REVIEW

File: snippets/language-css.cson [hunk 1]
```diff
@@ -17,21 +12,40 @@
     'body': '@font-face {\n\t$1\n}$0'
     'description': 'Specify online fonts to display text on their web pages.'
     'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face'
+  '@keyframes':
+    'prefix': 'keyframes'
+    'body': '@keyframes ${1:identifier} {\n\t$2\n}$0'
+    'description': 'Specifies a CSS animation.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en/docs/Web/CSS/@keyframes'
+   '@import':
```
Comment (50Wliu) [inline snippets/language-css.cson]: i comes before k in the alphabet :wink:

Comment (stowball) [inline snippets/language-css.cson]: ZOMG I've just fixed it and amended the commit >.<

**Comment by 50Wliu** [comment]
Thanks!

**Comment by stowball** [comment]
NP

**Pull request merged**